platform/darwin/src/MGLTypes.h

```diff
@@ -93,6 +93,10 @@ typedef struct __attribute__((objc_boxable)) MGLTransition {
     NSTimeInterval delay;
 } MGLTransition;
 
+NS_INLINE NSString *NSStringFromMGLTransition(MGLTransition transition) {
```

Comment:
Let's prefix this too.

Reply:
Although this is public I just added this function in this branch so we are fine changing the name.